superset/assets/stylesheets/superset.less

```diff
@@ -32,10 +32,6 @@ input.form-control {
     background-color: white;
 }
 
-input.form-control.input-sm {
```

Comment:
@graceguo-supercat this redefines the theme's input-sm from 30 to 36 which isn't ideal. Yet reverting this doesn't seem to affect react-select in a bad way... Do you more insight as to why this css entry is here?

Reply:
Agree. set input-sm from 30 to 36 isn't good. I also fixed in https://github.com/apache/incubator-superset/pull/3852